arviz/plots/backends/matplotlib/pairplot.py

```diff
@@ -58,12 +98,23 @@ def plot_pair(
                 ax=ax,
                 **plot_kwargs
             )
-        else:
+        elif kind == "hexbin":
             hexbin = ax.hexbin(
                 infdata_group[0], infdata_group[1], mincnt=1, gridsize=gridsize, **plot_kwargs
             )
             ax.grid(False)
 
+        else:
+            ax.plot(infdata_group[0], infdata_group[1], zorder=-1, **plot_kwargs)
+            plot_kde(
+                infdata_group[0],
+                infdata_group[1],
+                ax=ax,
+                contourf_kwargs={"alpha": 0},
+                contour_kwargs={"colors": "k"},
+                fill_last=False,
```

Comment:
Building on previous comment, I'd use setdefaults and provided fill_last (maybe default fill_last to None so we can use True or False depending on plot still allowing user to customize this). Something like:

```
plot_kwargs.setdefault("contour_kwargs", {})
plot_kwargs["contour_kwargs"].setdefault("alpha", 0)
...
plot_kde(
    ...,
    **plot_kwargs
)
```

If reusing code it may also be a good idea to move the setdefault part somewhere else.